irohad/torii/impl/command_service_impl.cpp

```diff
@@ -0,0 +1,226 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "torii/impl/command_service_impl.hpp"
+
+#include <thread>
+
+#include "ametsuchi/block_query.hpp"
+#include "builders/protobuf/transaction_responses/proto_transaction_status_builder.hpp"
+#include "common/byteutils.hpp"
+#include "common/is_any.hpp"
+#include "cryptography/default_hash_provider.hpp"
+#include "interfaces/iroha_internal/transaction_batch_factory.hpp"
+#include "interfaces/iroha_internal/transaction_sequence.hpp"
+#include "validators/default_validator.hpp"
+
+namespace torii {
+
+  CommandServiceImpl::CommandServiceImpl(
+      std::shared_ptr<iroha::torii::TransactionProcessor> tx_processor,
+      std::shared_ptr<iroha::ametsuchi::Storage> storage,
+      std::shared_ptr<iroha::torii::StatusBus> status_bus)
+      : tx_processor_(tx_processor),
+        storage_(storage),
+        status_bus_(status_bus),
```

Comment:
Maybe, you should move `tx_processor`, `storage` and `status_bus` here?

Reply:
not a big deal